extensions/adobe/experience/marketo/account-marketo.schema.json

```diff
@@ -0,0 +1,124 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/account-marketo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/account"],
+  "meta:extends": ["https://ns.adobe.com/xdm/context/account-crm"],
+  "description": "Marketo specific Account fields.",
+  "definitions": {
+    "account-marketo": {
+      "properties": {
+        "marketo:accountID": {
```

Comment:
Can you please update all the descriptions to be something other than a copy of the Ttile?

Reply:
I did updated the descriptions wherever I could. For properties where we did not have descriptions, I removed the descriptions. We will let Marketo add those as necessary as they own the "account-marketo" mixin.